detection_rules/beats.py

```diff
@@ -130,17 +135,29 @@ def get_beats_sub_schema(schema: dict, beat: str, module: str, *datasets: str):
     return {field["name"]: field for field in sorted(flattened, key=lambda f: f["name"])}
 
 
+def get_versions():
+    return [Version(f.lstrip('v').rstrip('.json.gz')) for f in os.listdir(get_etc_path("beats_schemas"))]
```

Comment:
I would be careful with using a long string with rstrip, it's very different from rstrip_suffix.

re.match, str.split, or str.partition might be better choices here. (EQL between() would be legit!)